keybinder3: 0.3.0 -> 0.3.2

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/keybinder3/versions.



meta.description for keybinder3 is: '"Library for registering global key bindings"'.

[Release on GitHub](https://github.com/kupferlauncher/keybinder/releases/tag/keybinder-3.0-v0.3.2)


[Compare changes on GitHub](https://github.com/engla/keybinder/compare/keybinder-3.0-v0.3.0...keybinder-3.0-v0.3.2)

<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.3.2 with grep in /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2
- directory tree listing: https://gist.github.com/5c2df9463b0fcd250df6c248d8e5f6de
- du listing: https://gist.github.com/96d6fc3b1001b76c6425c37bd80cd4d9

</details>
<details>
<summary>
Rebuild report (click to expand)
</summary>

46 total rebuild path(s)

15 package rebuild(s)

15 x86_64-linux rebuild(s)
15 i686-linux rebuild(s)
1 x86_64-darwin rebuild(s)
15 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
guake
kazam
keybinder3
kupfer
peek
quodlibet-full
quodlibet-xine-full
synapse
terminator
xfce.xfce4-pulseaudio-plugin
xfce.xfce4_pulseaudio_plugin
xfce4-12.xfce4-pulseaudio-plugin
xfce4-12.xfce4_pulseaudio_plugin
xfce4-13.xfce4-volumed-pulse
xfceUnstable.xfce4-volumed-pulse

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A keybinder3 https://github.com/r-ryantm/nixpkgs/archive/2481e214ceac693861856c4f69a5bbd8dbb81992.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2
ls -la /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2/bin
```


</details>
<br/>


cc @cstrahan for testing.

r-ryantm committed:
keybinder3: 0.3.0 -> 0.3.2

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/keybinder3/versions

GrahamcOfBorg commented:
<!--REQUEST_ID=3ca13f64-1ad1-4bde-8051-abb8540a48aa-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50718&attempt_id=471b8b1a-51bf-4abd-a0c5-723eda56e6d9)

The following builds were skipped because they don't evaluate on x86_64-darwin: keybinder3

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnsupportedSystem = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnsupportedSystem = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=3ca13f64-1ad1-4bde-8051-abb8540a48aa-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50718&attempt_id=d4f00bc2-0374-48ec-bb06-ebf55101bd84)

Attempted: keybinder3

<details><summary>Partial log (click to expand)</summary><p>

```
make[1]: Leaving directory '/build/source'
glibPreFixupPhase
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2
shrinking /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2/lib/libkeybinder-3.0.so.0.0.0
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2/lib
patching script interpreter paths in /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2
checking for references to /build in /nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2...
/nix/store/wav25zh29iwb6yk8alnmk3jdb5yl5wyb-keybinder3-0.3.2
```
</p></details>

c0bw3b committed:
keybinder3: refresh

Use new GitHub repo at https://github.com/kupferlauncher/keybinder
Move autoconf/automake/libtool to nativeBuildInputs

GrahamcOfBorg commented:
<!--REQUEST_ID=0cdbeb88-96aa-4a4e-a027-2abc243d9f1e-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50718&attempt_id=124d06dc-7541-4e50-b5db-074c655087a4)

The following builds were skipped because they don't evaluate on x86_64-darwin: keybinder3

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnsupportedSystem = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnsupportedSystem = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=0cdbeb88-96aa-4a4e-a027-2abc243d9f1e-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.50718&attempt_id=6347f0d0-bdb4-4bf8-a61d-f0eb50e33bc5)

Attempted: keybinder3

<details><summary>Partial log (click to expand)</summary><p>

```
make[1]: Leaving directory '/build/source'
glibPreFixupPhase
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2
shrinking /nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2/lib/libkeybinder-3.0.so.0.0.0
strip is /nix/store/6dpnd5aniypn8124mmy8f88s4mq2zl07-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2/lib
patching script interpreter paths in /nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2
checking for references to /build in /nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2...
/nix/store/0713xrslzmqfyd4spcgcg4qrljl46777-keybinder3-0.3.2
```
</p></details>
